metadata/animate.xml

```diff
@@ -116,5 +133,10 @@
 			<_long>Sets the color of the fire effects, alpha is ignored</_long>
 			<default>#b22303ff</default>
 		</option>
+		<option name="squeezimize_duration" type="animation">
+			<_short>Squeezimize duration</_short>
+			<_long>Sets the duration of the squeezimize animation in milliseconds.</_long>
+			<default>1000ms linear</default>
```

Comment:
I think this is way too slow, I would personally set it to 150ms :)

Reply:
Sure, I have no objection to tweaking the defaults.